### PR TITLE
[201811] Stop PMON docker before cold boot and soft reboot

### PR DESCRIPTION
**Pull request by aravindmani-1**
What I did
Stopped PMON before cold boot

How I did it
Edit reboot script and stop pmon docker and services

How to verify it
Check docker ps output just before cold boot is being executed.

Previous command output (if the output of a command-line utility has changed)
New command output (if the output of a command-line utility has changed)
UT:
[cold_boot_UT_201811.txt](https://github.com/Azure/sonic-utilities/files/6161410/cold_boot_UT_201811.txt)

### CODE REVIEW

**Comment by aravindmani-1** [comment]
@yxieca , @jleveque Can you please review this PR?.

**Review by jleveque — CHANGES_REQUESTED** [review]
Please also add the same change in the soft-reboot script

**Comment by aravindmani-1** [comment]
> Please also add the same change in the soft-reboot script

Added the same change in soft-reboot.

File: scripts/soft-reboot [hunk 1]
```diff
@@ -160,6 +172,5 @@ fi
 debug "Rebooting with ${REBOOT_METHOD} to ${NEXT_SONIC_IMAGE} ..."
 exec ${REBOOT_METHOD}
 
-
 VERBOSE=yes debug "Issuing OS-level reboot ..." >&2
-exec /sbin/reboot $@
+exec /sbin/reboot $@
```
Comment (jleveque) [inline scripts/soft-reboot:176]: Add newline back at EOF

**Review by jleveque — CHANGES_REQUESTED** [review]

File: scripts/soft-reboot (hunk 1 above)
Comment (aravindmani-1) [reply scripts/soft-reboot]: Done.

**Review by jleveque — APPROVED** [review]

**Review by yxieca — APPROVED** [review]

**Pull request merged**